presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -598,7 +598,14 @@ default void dropMaterializedView(ConnectorSession session, SchemaTableName view
 
     /**
      * Get the materialized view status to inform the engine how much data has been materialized in the view
+     * @param baseQueryDomain The domain from which to consider missing partitions. For example, a query that
+     * selects a specific date range can consider only partitions from that range when determining view staleness.
      */
+    default MaterializedViewStatus getMaterializedViewStatus(ConnectorSession session, SchemaTableName materializedViewName, TupleDomain<String> baseQueryDomain)
+    {
+        throw new PrestoException(NOT_SUPPORTED, "This connector does not support getting materialized views status");
+    }
+
     default MaterializedViewStatus getMaterializedViewStatus(ConnectorSession session, SchemaTableName materializedViewName)
```

Comment:
Just realized this: why do we still keep this interface? Can we fix presto-facebook as well to remove it completely?

Reply:
You can send out another PR to fix this BTW